columnflow/plotting/plot_functions_1d.py

```diff
@@ -145,12 +144,12 @@ def plot_shifted_variable(
     variable_settings: dict | None = None,
     **kwargs,
 ) -> plt.Figure:
-
-    hists = apply_variable_settings(hists, variable_insts, variable_settings)
+    """
+    TODO.
+    """
     variable_inst = variable_insts[0]
-
+    hists = apply_variable_settings(hists, variable_insts, variable_settings)
```

Comment:
`variable_inst = variable_insts[0]` just creates a pointer, right? Otherwise, the order here would be important since we also want to change `variable_inst`with the `apply_variable_settings` function.

Reply:
I see. Changed this :+1: